### PR TITLE
fix controller model visibility problem

### PR DESCRIPTION
**Pull request by NolaDonato**
1. call controller activation from framework thread
2. call bindShaders after controller model is added
3. remove unused assets

This fixes the problem with the controller model not becoming visible when the controller is activated after the app starts. gvr-controller exhibited this problem.

GearVRF DCO signed off by: Nola Donato nola.donato@samsung.com

### CODE REVIEW

**Comment by roshanch** [comment]
not sure if i messed up something with my repo, but it shows black screen with gvr-controller.

**Comment by liaxim** [comment]
Worked fine for me with Erik's app.

**Comment by roshanch** [comment]
@liaxim did you see the texture on the controller? I dont see it, but it fixes the issue mentioned.

File: GVRf/Framework/framework/src/main/java/org/gearvrf/io/GearCursorController.java [hunk 1]
```diff
@@ -206,7 +203,14 @@ public void onDrawFrame()
...
                 initialized = true;
-                context.getInputManager().activateCursorController(GearCursorController.this);
+                Runnable task = new Runnable()
+                {
+                    public void run()
+                    {
+                        context.getInputManager().activateCursorController(GearCursorController.this);
+                    }
+                };
+                context.runOnTheFrameworkThread(task);
```
Comment (liaxim) [inline GVRf/Framework/framework/src/main/java/org/gearvrf/io/GearCursorController.java]: Why the framework thread? It seems to me that all this adds is some latency. There is no other occasion the cursor controllers are using the framework thread. They probably should be for all callbacks but they don't. Also the callbacks right now are on a number of possible threads. Is this change mandatory for the fix to work?

**Comment by NolaDonato** [comment]
Doesn't onInit run on the framework thread? That is where the cursor controllers are activated except for the Gear which might not be ready then. The reason I made this change is that the cursor controller loads an asset which should probably not occurs on the GL thread.

**Comment by liaxim** [comment]
The cursor controllers are quite frivolous when it comes to threading. onInit does run on the framework thread, yes.

**Comment by liaxim** [comment]
Planning to merge on Tuesday. @roshanch ?

**Comment by liaxim** [comment]
Erik's app has been verified by QA.

**Comment by roshanch** [comment]
@liaxim again something wrong with controller, i don't see the controller in app. @NolaDonato is going to check it on her end.

**Comment by NolaDonato** [comment]
Will include this in upcoming controller reform

**Pull request closed without merging**